angel_system/ptg_eval/activity_classification/compute_scores.py

```diff
@@ -2,23 +2,23 @@
 from sklearn.metrics import average_precision_score
 import numpy as np
 
-log = logging.getLogger("ptg_eval")
+log = logging.getLogger("ptg_eval_activity")
 
 
 class EvalMetrics():
-    def __init__(self, labels, gt_true_mask, dets_per_valid_time_w, output_dir=''):
+    def __init__(self, labels, gt_true_mask, window_class_scores, output_dir=''):
         """
         :param labels: Array of class labels (str)
         :param gt_true_mask: Matrix of size (number of valid time windows x number classes) where True
```

Comment:
It is a little confusing, I think, to say size is (number of valid time windows x number classes) because valid is a little ambiguous thus far. Maybe (number time windows x number classes) and then explain that these should be valid time windows where scoring the classification there is valid.

Reply:
also, end sentence with period, I believe.